lib/duffel_api/services/aircraft_service.rb

```diff
@@ -22,13 +22,19 @@ def list(options = {})
       end
 
       # Returns an `Enumerator` which can automatically cycle through multiple
-      # pages of `Resources;:Aircraft`
+      # pages of `Resources::Aircraft`.
+      #
+      # By default, this will use pages of 200 results under the hood, but this
+      # can be customised by specifying the `:limit` option in the `:params`.
       #
       # @param options [Hash] options passed to `#list`, for example `:params` to
       #   send an HTTP querystring with filters
       # @return [Enumerator]
       # @raise [Errors::Error] when the Duffel API returns an error
       def all(options = {})
+        options[:params] ||= {}
+        options[:params] = DEFAULT_ALL_PARAMS.merge(options[:params])
```

Comment:
🎨  We could reduce to one-line if you want.

```suggestion
        options[:params] = DEFAULT_ALL_PARAMS.merge(options[:params] || {})
```

Reply:
This is a nice cosmetic improvement! I'll apply it across the code.